documentation/asciidoc/topics/con_operator_deployments.adoc

```diff
@@ -0,0 +1,22 @@
+//This topic is intended as downstream content.
+[id='infinispan-operator-deployments_{context}']
+= {ispn_operator} deployments
+
+When you install {ispn_operator}, it extends the Kubernetes API with Custom Resources (CR) for deploying and managing {brandname} clusters on {openshift}.
```

Comment:
CRs and CRDs are backwards here. CRDs extend the k8s API, CRs are what the user creates/applies

Reply:
facepalm.gif fixed, thanks.